turbo/stages/stageloop.go

```diff
@@ -308,8 +308,19 @@ func (h *Hook) AfterRun(tx kv.Tx, finishProgressBefore uint64) error {
 		if currentHeder.Number.Uint64() == 0 {
 			notifications.Accumulator.StartChange(0, currentHeder.Hash(), nil, false)
 		}
+		var pendingBlobFee uint64 = 1
+		excessBlobGas := misc.CalcExcessBlobGas(currentHeder)
+		if currentHeder.ExcessBlobGas != nil {
+			f, err := misc.GetBlobGasPrice(excessBlobGas)
```

Comment:
`currentHeader.ExcessBlobGas` should be already set at this point and there's no need to recalculate it. It's equal to `misc.CalcExcessBlobGas(parentHeader)`, not `misc.CalcExcessBlobGas(currentHeader)`.

Reply:
That's set for the current block's current blob fee, which the current block's transactions should abide by. 
`currentHeader.ExcessBlobGas` is set using the current block's parent header, and the BlobPrice per gas is directly calculated using that value. The accounting would then deduct the eventual cost using this price/fee value from the transactions in the block. Now the block is done. That whole validation is already done. 
When the next block is requested, it will be updated in the header then. We can't use the outdated (completed block's header's) value for transactions in the pool, we need the next block's blob fee. That's the "pending" blob fee.
Current-Header is the `parentHeader` for this.
What am I missing?

Reply:
Ah, yes, you're right, my bad.